packages/drivers/odsp-driver/src/buildOdspShareLinkReqParams.ts

```diff
@@ -0,0 +1,26 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ShareLinkTypes, SharingLinkKind } from "@fluidframework/odsp-driver-definitions";
+
+/**
+ * Build request parameters to request for the creation of a sharing link along with the creation of the file
+ * through the /snapshot api call.
+ * @param shareLinkType - Kind of sharing link reuested
+ * @returns A string of request parameters that can be concatenated with the base URI
+ */
+export function buildOdspShareLinkReqParams(shareLinkType: ShareLinkTypes | SharingLinkKind | undefined) {
```

Comment:
Maybe we can just add this function in odspUtils.ts(if expected to be used internally) or odspPublicUtils.ts(if expected to be used by partner also) as this is just another utility function instead of adding an additional file.